runtime/src/bank.rs

```diff
@@ -7129,6 +7129,13 @@ impl TransactionProcessingCallback for Bank {
             self.inspect_account_for_accounts_lt_hash(address, &account_state, is_writable);
         }
     }
+
+    fn get_epoch_stake(&self, vote_address: &Pubkey) -> u64 {
+        self.get_current_epoch_vote_accounts()
+            .get(vote_address)
+            .map(|(stake, _)| (*stake))
+            .unwrap_or(0)
+    }
```

Comment:
nit: I just had to fix this recently, so to avoid confusion, can you place this function right below `get_current_epoch_vote_accounts` and call it `get_current_epoch_vote_account_stake`? This way it's super obvious that it uses the right (leader schedule) epoch.

Reply:
I'll rename the function. I don't think it can be moved under `get_current_epoch_vote_accounts()`, since it needs to be in the `impl TransactionProcessingCallback for Bank` block. I think it's better to keep the whole callback implementation in a single block.

Reply:
Ohh whoops, I don't think I noticed it was in the trait. That's fine!